hil/config.py

```diff
@@ -8,13 +8,85 @@
 import ConfigParser
 import logging.handlers
 import importlib
+from schema import Schema, Optional
 import os
 import sys
+from urlparse import urlparse
 
 cfg = ConfigParser.RawConfigParser()
 cfg.optionxform = str
 
 
+def string_is_bool(option):
+    """Check if a string matches ConfigParser's definition of a bool"""
+    return option.lower() in ['true', 'yes', 'on', '1',
+                              'false', 'no', 'off', '0']
+
+def string_is_web_url(option):
+    """Check if a string is a valid web URL"""
+    url = urlparse(option)
+    if url.scheme == '' or url.netloc == '':
+        return False
```

Comment:
Curious whether we could just `return url.scheme != '' and url.netloc != '' `.

I know you may wanna return the `False` explicitly?

Reply:
Sure, that works fine.